mne_bids/read.py

```diff
@@ -185,6 +186,45 @@ def _handle_participants_reading(participants_fname, raw,
     return raw
 
 
+def _handle_scans_reading(scans_fname, raw, bids_path, verbose=False):
+    """Read associated scans.tsv and set meas_date."""
+    scans_tsv = _from_tsv(scans_fname)
+    fname = bids_path.fpath.name
+
+    if '_split-' in fname:
+        # for split files, scans only stores the filename without ``split``
+        extension = bids_path.fpath.suffix
+        bids_path.update(split=None, extension=extension)
+        fname = bids_path.basename
+    elif fname.endswith('.pdf'):
+        # for BTI files, the scan is an entire directory
+        fname = fname.split('.')[0]
+
+    # get the row corresponding to the file
+    # use string concatenation instead of os.path
+    # to work nicely with windows
+    data_fname = bids_path.datatype + '/' + fname
```

Comment:
wouldn't this work just as well?

```suggestion
    data_fname = bids_path.datatype / fname
```

on Linux, it does:
```Python
In [6]: a =pathlib.Path("hihho")

In [7]: b= "nono"

In [8]: a / b
Out[8]: PosixPath('hihho/nono')

```

Reply:
@sappelhoff  but `bids_path.datatype` is a string